src/pybind/mgr/hello/__init__.py

```diff
@@ -0,0 +1 @@
+from module import *  # NOQA
```

Comment:
@yaarith might want to change this to
```py
from .module import Module
```

as it's compatible with py3. and is more explicit in the sense that all we need is just the Module class.